### ISSUE TITLE
Test set

### ISSUE THREAD

**Issue by chesterkuo** [DRCSolutionService/DRCD#3]
What's current F1 score or EM score for test set if Chinese MRC dataset here ??

**Comment by DRCKnowledgeTeam** [comment]
As mentioned in the paper, we adjust https://github.com/YerevaNN/R-NET-in-Keras and train on DRCD 
F1 score on test set is 44.4 and exact match score is  29.05
